docs/en/MatrixOne/Overview/matrixcube/matrixcube-introduction.md

```diff
@@ -14,31 +14,31 @@ There are several key concepts for understanding how MatrixCube works.
 
 ### **Store**
 
-A MatrixCube distributed system consists of several physical computers, our data are stored across these physical computers. We call each computer inside this cluster a `Store`.
+A MatrixCube distributed system consists of several physical computers; our data is stored across these physical computers. We call each computer inside this cluster a `Store`.
 
 ### **Shard**
 
-Our data in database are organized in tables logically. But for physical storage, the data are split into different partitions to store in order to get better scalability. Each partition is called a `Shard`. In our design, a new created table is initially a `Shard`. When the size of the table exceeds the `Shard` size limit, the `Shard` will split. 
+Our data in the database is organized in tables logically. But for physical storage, the data is split into different partitions to store in order to get better scalability. Each partition is called a `Shard`. In our design, a new created table is initially a `Shard`. When the size of the table exceeds the `Shard` size limit, the `Shard` will split. 
 
 ### **Replica**
 
-To provide reliable service, each `Shard` is stored not only once, it will have several copy stored in different `Stores`. We call each copy a `Replica`. A `Shard` can have multiple `Replica`, the data in each `Replica` are the same. 
+To provide reliable service, each `Shard` is stored not only once, but will have several copy stored in different `Stores`. We call each copy a `Replica`. A `Shard` can have multiple `Replica`, the data in each `Replica` is the same. 
 
 ### **Raft-group and Leader**
 
-Since multiple `Replicas` are located in different `Stores`, once a `Replica` is updated, the other `Replicas` must be updated to keep data consistency. When a client makes query to no matter which `Replica`, it always gets the same result. We deploy Raft protocol to implement the concensus process. The `Replicas` of a particular `Shard` group into a `Raft-group`. 
+Since multiple `Replicas` are located in different `Stores`, once a `Replica` is updated, the other `Replicas` must be updated to keep data consistency. When a client makes a query to any `Replica`, it always gets the same result. We deploy Raft protocol to implement the concensus process. The `Replicas` of a particular `Shard` will group into a `Raft-group`. 
```

Comment:
Since multiple `Replicas` are located in different `Stores`, once a `Replica` is updated, the other `Replicas` must be updated to keep data consistency (which the system does automatically).